kremlib/c/fstar_uint128_struct_endianness.h

```diff
@@ -0,0 +1,59 @@
+/* Verified uint128 implementation. */
```

Comment:
also please update the comment: hand-written implementation of endianness-related uint128 functions for the extracted uint128 implementation